### PR TITLE
Mc 12986 create network policy

### PR DESCRIPTION
**Pull request by y0geshdev**
#### Changes made
Added API docs for creating network policy rules.

### CODE REVIEW

File: source/includes/stackpath/_network_policy_rules.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+<!-------------------- CREATE A NETWORK POLICY RULE -------------------->
+
+### Create a network policy rule
+
+```shell
+curl -X POST \
+  -H "MC-Api-Key: your_api_key" \
+  -d "request_body" \
+  "https://cloudmc_endpoint/v1/services/stackpath/test-area/networkpolicyrules?workloadId=bf9fd2ac-f761-46ef-88e0-b61ef68f8619"
```
Comment (razine-bensari) [inline source/includes/stackpath/_network_policy_rules.md:14]: Isn'it supposed to have a backslash like so:
`https://cloudmc_endpoint/v1/services/stackpath/test-area/networkpolicyrules/?workloadId=bf9fd2ac-f761-46ef-88e0-b61ef68f8619`

As per line 29?

Comment (razine-bensari) [reply source/includes/stackpath/_network_policy_rules.md:14]: networkpolicyrules/?workloadId
or
networkpolicyrules?workloadId

the latter makes more sense but both are present in the docs.

**Review by razine-bensari — CHANGES_REQUESTED** [review]
Nice!, one small typo I think deserves to be addressed, otherwise nice job!

File: source/includes/stackpath/_network_policy_rules.md (hunk 1 above)
Comment (y0geshdev) [reply source/includes/stackpath/_network_policy_rules.md:14]: Yes, it should be `networkpolicyrules?workloadId`.
Fixed in 3f79950

**Review by razine-bensari — APPROVED** [review]

**Review by patrickspensieri — APPROVED** [review]

**Pull request merged**